src/utils.jl

```diff
@@ -2,6 +2,52 @@ export subscript, superscript
 export record_interaction
 export rotate2D, scale, Polygon, Point2f0
 
+##########################################################################################
+# Check/get/set
+##########################################################################################
+
+"""
+    has_value(p, key)
```

Comment:
More naming gripe. This function should be named `has_key`, as that's what it checks :P

Reply:
I thought about that as well but I didn't like it because we don't only check for keys (in case of Dict) but also for fieldnames (in case of structs). Surely the meaning of `key` is broad enough that it is usable in this context. Maybe you have any better idea?

Reply:
`has_key` is good I think.

Reply:
ok done